contributors/guide/OWNERS

```diff
@@ -1,13 +1,10 @@
 # See the OWNERS docs at https://go.k8s.io/owners
 
-reviewers:
-  - castrojo
-  - guineveresaenger
-  - idvoretskyi
-  - tpepper
```

Comment:
@idvoretskyi , @tpepper 

Would you like to remain as reviewers in the contributor guide owners file?